mods/rv/rules/ai.yaml

```diff
@@ -639,6 +639,8 @@ Player:
 		ConstructionYardTypes: gacnst, nacnst, yacnst
 		RefineryTypes: garefn, garefb, garefc, garefd, narefn, narefb, narefc, narefd, yarefn, yarfn2, yarfnb, yarfnc, yarfnd
 		SiloTypes: gasilo, nasilo, yasilo
+		AdditionalMinimumRefineryCount: 0 ## We don't need this
+		InititalMinimumRefineryCount: 0 ## We don't need this
```

Comment:
I don't think this is a good idea, i had did a quick test and had an AI end up getting to 0 cash before they build a refinery. 
![resim](https://user-images.githubusercontent.com/7933210/191297773-b2a9df48-7077-4591-b9b8-70fb9cdde3cf.png)

It is needed for the cleanup i think tho, so i think it is better we keep things as is for now. Just do the crash fix.

Reply:
I will add a 100% build for refinery for it then, like what I do for SP

Reply:
Fixed. Now AI will also build barrack first to prevent player infantry rush at early game.

Now AI will build powerplant (may build extra), barrack (may build extra), refinary

Reply:
In my opinion, `AdditionalMinimumRefineryCount` and `InititalMinimumRefineryCount` is too hardcoded and has bug on muti-queues